### PR TITLE
ESM support, typings, distributable bundle size reduce

### PR DESCRIPTION
**Pull request by xobotyi**
* Added supportd of ES modules (isMobile is exported as default) 0089dcb16760914f5db0b59010b1653b43d33d99
This will allow to write 
  ```javascript
  import isBrowser from "is-browser";
  ```
  instead of 
  ```javascript
  import  *  as isBrowser from "is-browser";
  ```
* Actualized `index.d.ts` for TS users code assitance 3f2b7bbf158840e8912e667fade37b1ae07b8a81
* Reduced distributable bundle size e43ed8cf0cf3e43ac93f5f9514170d9aac3fff94

### CODE REVIEW

File: index.es.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict'
```
Comment (juliangruber) [inline index.es.js]: I would rather not maintain a `index.es.js` and a `index.js`, can we reduce it to just one file, by changing the export in `index.js` to

```js
module.exports = module.exports.default = isMobile
```

File: index.d.ts [hunk 2]
```diff
@@ -1 +1,16 @@
-export declare function isMobile(ua?: string|object, opts?: object): boolean;
```
Comment (juliangruber) [inline index.d.ts]: is this going to be a breaking change?

File: index.es.js (hunk 1 above)
Comment (xobotyi) [reply index.es.js]: Actually that would be even better =)
I'll edit the commit

File: index.d.ts (hunk 2 above)
Comment (xobotyi) [reply index.d.ts]: Sure! Here described two optional parameters, but function expecting an optional object, with two optional parameter one of which is node's request object.
Long things short - mine is the poper one (and declared in package.json - it is necessary to make typings work for external modules) %)

Comment (juliangruber) [reply index.d.ts]: @egorovsa you had initially created the typedef, are you cool with this?

**Comment by xobotyi** [comment]
@juliangruber shall i close the PR? =)
It stuck here for half a year already.

**Comment by juliangruber** [comment]
Sorry, I was waiting for @egorovsa to reply. Could you resolve the conflict in package-lock.json? Then I will merge :)

**Comment by xobotyi** [comment]
Suddenly i've removed the forked repo =)
I'll create a new PR

**Pull request closed without merging**